### ISSUE TITLE
Clicking toolbar button has no effect.

### ISSUE THREAD

**Issue by rainwoodman** [bokeh/bokeh#4623]
Firefox 47.0; following the datashader example.

For defects or deficiencies, please provide:
* Description of expected behavior and the observed behavior

Expected behavior: after clicking a toolbar button it is underlined with blue.

Observed behavior: the button is not underlined with blue. Therefore, I can not switch the tool from pan to zoom.

* Self-contained example code that generates the issue (if possible)
```
from bokeh.plotting import output_notebook, figure, show

output_notebook()

from IPython.core.display import HTML, display
display(HTML("<style>.container { width:90% !important; }</style>"))
p = figure(x_range=(0, 360), y_range=(-10, 40), tools="pan,box_zoom,reset")
show(p)
```
* Stack traceback and/or browser developer tools console output
```
Bokeh: all callbacks have finished main.min.js:14:5
Kernel: kernel_connected (bbec776a-7a56-42a0-9015-d4e039bd178f) main.min.js:22356:13
Kernel: kernel_ready (bbec776a-7a56-42a0-9015-d4e039bd178f) main.min.js:22356:13
Bokeh: BokehJS loaded, going straight to plotting main.min.js:74:7
Bokeh: BokehJS loaded, going straight to plotting main.min.js:74:5
Bokeh: setting log level to: 'info' bokeh-0.11.1.min.js:2:29515
"Bokeh: injecting CSS: https://cdn.pydata.org/bokeh/release/bokeh-0.11.1.min.css" main.min.js:60:7
"Bokeh: injecting CSS: https://cdn.pydata.org/bokeh/release/bokeh-widgets-0.11.1.min.css" main.min.js:62:7
Bokeh: BokehJS loaded, going straight to plotting main.min.js:74:7
Bokeh: Registering Jupyter comms for target a58609c6-7746-415f-bb3a-9c42da088e73 bokeh-0.11.1.min.js:62:11364
Bokeh: BokehJS loaded, going straight to plotting main.min.js:74:5
Bokeh: setting log level to: 'info' bokeh-0.11.1.min.js:2:29515
"Bokeh: injecting CSS: https://cdn.pydata.org/bokeh/release/bokeh-0.11.1.min.css" main.min.js:60:7
"Bokeh: injecting CSS: https://cdn.pydata.org/bokeh/release/bokeh-widgets-0.11.1.min.css" main.min.js:62:7
Bokeh: BokehJS loaded, going straight to plotting main.min.js:74:7
Bokeh: Registering Jupyter comms for target 0fe5f628-46ca-4d52-a77f-3aa36645ff1e
```
* Software version information output by `bokeh info` command
```
Python version      :  3.5.1 |Anaconda custom (64-bit)| (default, Dec  7 2015, 11:16:01)                                                  
IPython version     :  4.2.0                                                                                                              
Bokeh version       :  0.11.1                                                                                                             
BokehJS static path :  /anaconda3/lib/python3.5/site-packages/bokeh/server/static 
```
For desired features, please provide:
* Description of the desired feature

**Comment by rainwoodman** [comment]
The problem is only in triggering the tool. If I put box_zoom as the first tool, I can use zoom. If I put pan as the first tool, I can pan. But I cannot select any other tools.

**Comment by birdsarah** [comment]
Are you by any chance on a laptop with a dual touch screen and mouse?

**Comment by birdsarah** [comment]
If yes, this should have been fixed in 0.12 which is coming out very shortly, and you can use the dev builds. Or if you don't want to use a dev build: see http://bokehplots.com/pages/faqs.html

If no, then we have a problem.

**Comment by rainwoodman** [comment]
I confirm this only happens on my laptop which has touch screen and a trackpoint (mouse). Thanks. This must be a duplicate. If 0.12 still has the issue I will reopen. Thanks for the quick response!
